vowels.py

```diff
@@ -0,0 +1,28 @@
+# Automation onboarding
+# Write a program to find the number of vowels and consonants in a given string.
+# Vowels: a, e, i, o, u
+import re
+
+def vowels(input):
+    #print (input)
+    # define a list to store all vowels and all consonants
+    vowelList = []
+    consonantList = []
+
+    # remove any special characters and space
+    input2 = re.sub('[^\w]+|_', '', input)
```

Comment:
It nice and at the same time fancy using regex, good job on that 💪 .
try to use representative names for variables, input2 is not representative :)

Reply:
Good point! 
Changed all "input2" to "input_result"